shared/metrics/src/main/java/io/pravega/shared/metrics/MetricsConfig.java

```diff
@@ -174,7 +174,7 @@
     private MetricsConfig(TypedProperties properties) throws ConfigurationException {
         this.enableStatistics = properties.getBoolean(ENABLE_STATISTICS);
         this.dynamicCacheSize = properties.getLong(DYNAMIC_CACHE_SIZE);
-        this.dynamicCacheEvictionDurationMs = Duration.ofMinutes(properties.getInt(DYNAMIC_CACHE_EVICTION_DURATION_MINUTES)).toMillis();
+        this.dynamicCacheEvictionDurationMinutes = Duration.ofMinutes(properties.getInt(DYNAMIC_CACHE_EVICTION_DURATION_MINUTES)).toMillis();
```

Comment:
You use `toMillis` here, but assign it to a field named `Minutes`.This looks very wrong.

Reply:
If you are fixing this class, please let's get out of the dark ages and use `Duration` instead of some `long` field that contains some value for which nobody knows what the unit is. All other config classes that use time express this as a `Duration` (or at least they should be).

Reply:
You are right, using `Duration` instead.